package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-to-print",
-  "version": "2.0.0-alpha-4",
+  "version": "2.0.0-alpha-5",
```

Comment:
Nit: we should bump the version in a separate commit

Reply:
This is due to the package version being bumped for release but not running `npm install` after. I can remove that change though from when I ran `npm install`

Yet another one of my gripes about the npm lockfile. :shrug:

Reply:
Oh, I totally thought this was in `package.json` heh. Seems like we should do this, so just leave it in 👍